runtime/syntax/default.yaml

```diff
@@ -0,0 +1,16 @@
+filetype: unknown
+
+detect:
+    filename: ""
+
+rules:
+    # Mails
+    - special: "[[:alnum:].%_+-]+@[[:alnum:].-]+\\.[[:alpha:]]{2,}"
+    # URLs
+    - identifier: "(https?|ftp|ssh)://\\S+\\.\\S+[^])>\\s,.]"
+
+    - comment:
+        start: "(^|\\s)#"
```

Comment:
For the record, I'm still hesitant about this. The `#` can have completely different meanings in different files.